### ISSUE TITLE
Tag tarballs are broken

### ISSUE THREAD

**Issue by mbanck** [libmbd/libmbd#54]
Downloading and building the automatic tarballs associated with tags leads to cmake failures, because `cmake/libMBDVersionTag.cmake` is not included:

```
-- Detecting C compile features - done
CMake Error at cmake/libMBDVersion.cmake:12 (message):
  Not in a Git repository and version tag is missing, you most likely attempt
  to install from a copy of the source tree.  Obtain the source distribution
  (libmbd-<version>.tar.gz) from a Github release page instead.
Call Stack (most recent call first):
  CMakeLists.txt:13 (include)
```

Wouldn't it be possible to make a branch for release, add `cmake/libMBDVersionTag.cmake`  there and then tag so this works?

**Comment by mbanck** [comment]
Ok, looks like a duplicate of #28 - I disagree with the assertion there that the git-archive source code associated with a tag should not be suitable for distribution - for most projects on Github it is.

But I will try to work-around this in Debian/Ubuntu.

**Comment by mbanck** [comment]
Wait, you are releasing both a Fortran90 libmbd and a python pymbd with separate tarballs for each tag?

How are downstream distributions supposed to package this? What is the problem with just shipping one useful tarball and the user/developer/distributor/packager can build from it what they want?

**Comment by jhrmnn** [comment]
Hi, sorry for the late reaction. It seems like in the meantime you have figured out to use the correct tarballs, so I'll close this issue.

As for the distinction between a source tree and a source distribution, I'll refer for example to [PEP 517](https://peps.python.org/pep-0517/#source-distributions).

**Comment by mbanck** [comment]
I do not think it is resolved. Right now I need to manually patch in `cmake/libMBDVersionTag.cmake` everytime which I don't feel is a reasonable work-around.

I think if you make a release of the code on Github, there should be a corresponding release tarball that is self-contained and buildable. I understand that you consider PIP the more important distribution channel, but libMBD is being used in several other computational packages and it would be useful if there was an easily consumable release tarball for Linux distributions that want to ship those packages.

Right now, packaging of the current Quantum Espresso version on Debian/Ubuntu is blocked on libMBD (well, except I can hack around it for now).

**Comment by mbanck** [comment]
I do not think it is resolved. Right now I need to manually patch in `cmake/libMBDVersionTag.cmake` everytime which I don't feel is a reasonable work-around.

I think if you make a release of the code on Github, there should be a corresponding release tarball that is self-contained and buildable. I understand that you consider PIP the more important distribution channel, but libMBD is being used in several other computational packages and it would be useful if there was an easily consumable release tarball for Linux distributions that want to ship those packages.

Right now, packaging of the current Quantum Espresso version on Debian/Ubuntu is blocked on libMBD (well, except I can hack around it for now).

**Comment by jhrmnn** [comment]
But why can't you use the release source distribution tarballs, which are self-contained?
<img width="825" alt="image" src="https://github.com/libmbd/libmbd/assets/940353/635f477c-6bd6-42da-ab9a-67012e75700c">
